action.yml

```diff
@@ -289,12 +288,19 @@ runs:
       run: |
         composer run build:js:storefront
 
+    - name: Copy assets
+      if: inputs.install-storefront == 'true' || inputs.install-admin == 'true'
+      shell: bash
+      working-directory: ${{ inputs.path }}
+      run: |
+          bin/console assets:install
+
     - name: Assign default Storefront theme to all sales channels
       if: inputs.install-storefront == 'true'
       shell: bash
       working-directory: ${{ inputs.path }}
       run: |
-        if bin/console theme:change --help | grep -q '--sync'; then
+        if bin/console theme:change --help | egrep -q '^\s+--sync'; then
```

Comment:
This would produce a syntax error all the time (because grep interpreted the `--sync` as a flag), leading to an asynchronous build.